homeassistant/components/homeassistant_yellow/__init__.py

```diff
@@ -33,6 +34,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.async_create_task(hass.config_entries.async_remove(entry.entry_id))
         return False
 
+    if not await async_validate_hardware_consistent(hass):
+        ir.async_create_issue(
+            hass,
+            domain=DOMAIN,
+            issue_id=ISSUE_CM4_UNSEATED,
+            is_fixable=False,
+            is_persistent=True,
```

Comment:
Why do we set the persistent flag to `True`? Is the idea that if we detect an issue once, we want the user to reseat the CM4?

Reply:
Good point, I think I will remove it.